tests/test_pdf.py

```diff
@@ -173,7 +173,7 @@ def test_pdf_integration_staterror(backend):
     pdf = pyhf.Model(spec)
     par = pdf.config.par_slice('stat_firstchannel')
     par_set = pdf.config.param_set('stat_firstchannel')
-    tensorlib, _ = backend
+    tensorlib, _ = pyhf.get_backend()
```

Comment:
this should not have been changed. We have to inject the backend this way for testing purposes (if the tests are in weird orders, the `pyhf.get_backend()` isn't properly reset for tests).